internal/repotesting/repotesting.go

```diff
@@ -27,11 +27,18 @@ type Environment struct {
 	connected  bool
 }
 
+// Options used during Environment Setup.
+type Options struct {
+	NewRepositoryOptions func(*repo.NewRepositoryOptions)
+	OpenOptions          func(*repo.Options)
```

Comment:
The motivation is to be able to pass `*repo.Options` to inject time in another test harness.